host-interaction/process/inject/use-pe-injection.yml

```diff
@@ -0,0 +1,29 @@
+rule:
+  meta:
+    name: use pe injection
+    namespace: host-interaction/process/inject
+    author: 0x534a@mailbox.org
+    scope: function
+    att&ck:
+      - Defense Evasion::Process Injection::Portable Executable Injection [T1055.002]
+    examples:
+      - ce8d7590182db2e51372a4a04d6a0927a65b2640739f9ec01cfd6c143b1110da:0x4014E0
+    references:
+      - https://www.elastic.co/de/blog/ten-process-injection-techniques-technical-survey-common-and-trending-process
+  features:
+    - and:
+      - characteristic: loop
+      - optional:
+        - or:
+          - api: kernel32.OpenProcess
+          - api: ntdll.NtOpenProcess
+          - api: ntdll.ZwOpenProcess
```

Comment:
new (library) rule candidate: `open process`

Reply:
Fixed in aa94a33